runtime/near-vm-runner/Cargo.toml

```diff
@@ -14,8 +14,9 @@ This crate implements the specification of the interface that Near blockchain ex
 
 [dependencies]
 cached = "0.12"
-wasmer-runtime = { version = "=0.17.1", features = ["default-backend-singlepass"], default-features = false }
-wasmer-runtime-core = { version = "=0.17.1" }
+wasmer = { version = "=1.0.0-alpha01.0", features = ["singlepass"], default-features = false }
+wasmer-compiler = { version = "=1.0.0-alpha01.0", features = ["translator"] }
+wasmer-types = { version = "=1.0.0-alpha01.0" }
```

Comment:
Maybe make separate VMKind for 0.x and 1.0 versions of Wasmer?

Reply:
Sounds good, We'll have three VMKinds!

Reply:
I tried wasmer 1.0 can be compiled with stable rust. However, 0.17.1 cannot. Since this to support both wasmer 0.x and 1.0 we need to stick with nightly rust. Do you think it worth drop wasmer 0.x to use stable rust?